client/signup/style.scss

```diff
@@ -3,6 +3,15 @@
 	.layout__content {
 		padding: 32px 0 0 0;
 	}
+
+	//Masterbar is hidden but still has height
+	//which is how sticky panel offset is calculated.
+	//Setting height to zero removes the offset
+	//so the sticky panel sticks to the top.
+
+	&.has-no-masterbar .masterbar {
+		height: 0;
```

Comment:
@sixhours Does this only affect signup? Should this `height: 0` go in the masterbar.scss file instead?

Reply:
Chatting in Slack, we determined this should stay in signup for now; we don't use this method of hiding the masterbar elsewhere except in Jetpack Connect.